code/modules/admin/panicbunker.dm

```diff
@@ -0,0 +1,37 @@
+var/list/panic_bunker_bypass = list()
+
+/datum/admins/proc/panicbunker()
+	set category = "Server"
+	set name = "Toggle Panic Bunker"
+
+	if(!check_rights(R_SERVER))
+		return
+
+	config.panic_bunker = !config.panic_bunker
+
+	log_and_message_admins("[key_name(usr)] has toggled the Panic Bunker, it is now [(config.panic_bunker ? "on" : "off")]")
+	if (config.panic_bunker && (!dbcon || !dbcon.IsConnected()))
+		message_admins("The SQL database is not connected, player age cannot be checked and the panic bunker will not function until the database connection is restored.")
+
+/datum/admins/proc/addbunkerbypass(ckeytobypass as text)
+	set category = "Server"
+	set name = "Add Panic Bunker Bypass"
+	set desc = "Allows a given ckey to connect despite the panic bunker for a given round."
+
+	if(!check_rights(R_SERVER))
+		return
+
+	global.panic_bunker_bypass |= ckey(ckeytobypass)
+
+	log_admin("[key_name(usr)] has added [ckeytobypass] to the current round's bunker bypass list.")
+	message_admins("[key_name_admin(usr)] has added [ckeytobypass] to the current round's bunker bypass list.")
```

Comment:
Replace both log lines above with
`log_and_message_admins("has added [ckeytobypass] to the current round's bunker bypass list.")`